falkordb-cluster/cluster-entrypoint.sh

```
@@ -328,6 +328,26 @@ if [[ $RUN_METRICS -eq 1 ]]; then
   redis_exporter_pid=$!
 fi
 
+reconnect_to_master(){
+  res=$(redis-cli $AUTH_CONNECTION_STRING $TLS_CONNECTION_STRING info replication)
+  if [[ -n $res ]];then
+    master_info=$(redis-cli $AUTH_CONNECTION_STRING $TLS_CONNECTION_STRING info replication | grep role)
+    if [[ $master_info == *"role:slave"* ]];then
+      ip="$(echo "$res" | grep master_host | cut -d':' -f2 | tr -d '[:space:]')"
```

Shellcheck (0.10.0)

[warning] ip appears unused. Verify use (or export if used externally).

(SC2034)